tonic/src/transport/server/mod.rs

```diff
@@ -621,13 +646,26 @@ impl<L> Server<L> {
     }
 }
 
+async fn optinal_sleep_future(sleep_time: Option<Duration>) {
+    let _result = match sleep_time {
+        Some(time) => {
+            let sleep = tokio::time::sleep(time);
+            sleep.await;
+        }
+        None => {
+            future::pending::<()>().await;
+        }
+    };
+}
```

Comment:
Nits: spell it (as much as possible -- importing `sleep()` and `pending()`) as below, move below `serve_connection()`.

```rust
async fn sleep_or_pending(wait_for: Option<Duration>) {
    match wait_for {
        Some(wait) => sleep(wait).await,
        None => pending().await,
    }
}
```

Reply:
Fixed!